about/policy/sgid/index.md

```diff
@@ -1,116 +1,171 @@
 
-We will periodically review the SGID for datasets that no longer meet these criteria. Specifically, we may consider removing datasets that see little to no usage for a prolonged period of time.
+### General Qualifications
+{: .text-left}
 
-### Dataset Introduction Process
-{: .text-left }
+These criteria apply to all datasets added to the SGID, regardless of which part of the SGID they belong to.
 
-We will use the [Porter](https://github.com/agrc/porter) project to guide and track the steps needed to add or make breaking changes (remove, schema change, etc.) to products contained in the SGID. A brief overview of these steps is included below.
+- Datasets must cover a **state-wide** geographic extent, be part of a state-wide project or initiative (such as roads, address points, and parcels), or be relevant to broader projects or mandates that have a state-wide interest (e.g. geologic hazard mapping or endangered species mapping).
+- The entity providing the dataset must be the **authoritative** or original source (for example, a dataset of geologic hazards as identified by the Utah Geological Survey must come from the UGS, not another organization).
+- Datasets should have a reasonable **update schedule** included in the metadata.
+  - We may add datasets that result from a one-time project and won't be updated in the future, but this should be clearly stated in the metadata.
+- Datasets must not contain any information considered `private`, `protected`, or `controlled` under the Utah [Government Records Access and Management Act (GRAMA)](https://archives.utah.gov/opengovernment/) or any other personally identifiable information (PII).
 
-We will add a corresponding record to the [Data Stewardship spreadsheet](https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1). This will contain at a minimum:
+### SGID Index Qualifications
+{: .text-left}
 
-- The authoritative access from location
-- The table name
-- A description
-- The data steward
-- The steward contact name and email
-- A link to the data page on the AGRC website
-- A link to the SGID Open Data page
+These criteria apply to all datasets added to the SGID Index and not anywhere else in the SGID (for example, download links hosted on an entity's own server).
 
-We will create a hosted feature service of the dataset in our ArcGIS Online organization. The ArcGIS Online item for this feature service will reside in the corresponding category folder and group and contain the appropriate tags. If the dataset is authoritative, it will be marked as `authoritative`.
+- Datasets should be **publicly available** without requiring a login.
+  - We may add references to datasets that are restricted due to safety or security concerns as a way to raise awareness that the datasets exist but are restricted.
+- Dataset links should be **stable** and maintained for the forseeable future. We do not want to be constantly updating the Index due to dead links.
```

Comment:
I feel like we could omit the last sentence ("We do not want to be constantly updating the Index due to dead links.") and still get our point across.  Or maybe lighten it and say something like "this allows us to serve a reliable index"... something like that.